src/emulator/types.ts

```diff
@@ -36,7 +36,12 @@ export const ALL_SERVICE_EMULATORS = [
   Emulators.LOGGING,
 ];
 
-export const EMULATORS_SUPPORTED_BY_GUI = [Emulators.DATABASE, Emulators.LOGGING];
+export const EMULATORS_SUPPORTED_BY_GUI = [
```

Comment:
Do we need `LOGGING` in here?  Is there any meaning to this (below)?
```
firebase emulators:start --only logging
```

Reply:
no you're right, I've dropped `logging` from being enablable via `--only` so it doesn't need to be here